src/modules/commander/Commander.cpp

```diff
@@ -1959,16 +1959,21 @@ Commander::run()
 
 		if (_esc_status_sub.updated()) {
 			/* ESCs status changed */
-			esc_status_s esc_status{};
+			esc_status_check();
 
-			if (_esc_status_sub.copy(&esc_status)) {
-				esc_status_check(esc_status);
+		} else if ((_esc_connectiontype == esc_status_s::ESC_CONNECTION_TYPE_DSHOT)
```

Comment:
I added this line here because not all boards support DShot (including SITL). Does CAN ESCs or any other also have telemetry?

Reply:
Yes, it's the norm with UAVCAN ESCs.